source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
+Relational Migrator displays diagram names on the ribbon tab. 
+
+About this Task
+---------------
+
+You can rename diagrams to customize your diagram properties. Relational
+Migrator shows your diagram names on the ribbon tab of the Mapping view and in
+the Diagram Properties pane. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename. 
+
+      Click the name of the diagram in the ribbon tab to display the
+      diagram. 
+   
+   .. step:: Open the Diagram Properties pane.
+
+      Click the arrow button on the right of your window to open the
+      Diagram Properties pane. 
+
+      .. image:: /img/diagrams/diagram_properties.png
+         :alt: Diagram Properties pane
+      
+      The Diagram Properties pane displays the active diagram name and
+      notes. By default, Relational Migrator names diagrams in the order
+      they were created and leaves the notes field empty. 
+
+   .. step:: Enter your new diagram name in the Name box. 
+
+      Relational Migrator automatically saves your new diagram name and
+      updates the ribbon tab accordingly.
```

Comment:
[nit] for brevity, I think we can take out "accordingly"
```suggestion
      updates the ribbon tab.
```

Reply:
Done!